t/unit/tasks/test_tasks.py

```diff
@@ -251,6 +268,16 @@ def test_autoretry(self):
         self.autoretry_task.apply((1, 0))
         assert self.autoretry_task.iterations == 6
 
+    @patch('random.randrange')
+    def test_autoretry_backoff(self, randrange):
```

Comment:
If you could add another test case that verifies the jitter flow it would be nice!